internal/core/software_renderer.rs

```diff
@@ -297,6 +306,9 @@ impl Renderer for SoftwareRenderer {
         for item in items {
             item.cached_rendering_data_offset().release(&mut self.partial_cache.borrow_mut());
         }
+        // We don't have a way to determine the screen region of the delete items, what's in the cache is relative. So
+        // as a last resort, refresh everything.
+        self.force_screen_refresh.set(true);
```

Comment:
I would have tried to put the whole screen in `force_dirty` instead of adding a new variable

Reply:
That was my initial thinking, too, but I don't know how to retrieve that at this point when it's a parameter (indirectly) to render() for example. I could store that as `last_screen_size`, but that seems wasteful compared to a boolean.